### PR TITLE
Getting started guide for NetworkPolicy

### PR DESCRIPTION
**Pull request by caseydavenport**
Walks users through a simple example demonstrating NetworkPolicy.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1179)

### CODE REVIEW

**Comment by jaredbhatti** [comment]
Is this tied to release 1.4?  If not, can you make the PR against the master?

File: docs/getting-started-guides/network-policy.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```shell
+export NETWORK_POLICY_PROVIDER=calico
+curl -sS https://get.k8s.io | bash
+```
+
+## Using NetworkPolicy 
+
+Once your cluster is started, you should see a collection of pods running in the `kube-system` Namespace that support Kubernetes NetworkPolicy.
+
+```shell
```
Comment (errordeveloper) [inline docs/getting-started-guides/network-policy.md]: I think you can use `console` highlighter here instead, I've seen it used in other docs.

File: docs/getting-started-guides/network-policy.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Kubernetes can be used to declare network policies which govern how Pods can communicate with each other.  This document helps you get started using the Kubernetes [NetworkPolicy API](/docs/user-guide/networkpolicies), and provides a demonstration thereof. 
+
+In this article we assume that that a Kubernetes cluster has been created with network policy support.  It's easy to create such a cluster using the [GCE guide](/docs/getting-started-guides/gce), setting the `NETWORK_POLICY_PROVIDER=calico` environment variable.
+
+e.g
+
+```shell
+export NETWORK_POLICY_PROVIDER=calico
+curl -sS https://get.k8s.io | bash
```
Comment (errordeveloper) [inline docs/getting-started-guides/network-policy.md]: As someone working on next-generation provisioning/bootstrap tool, you are already linking to GCE guide above, does this have to be here really? We will have to figure out how to propagate reference to new bootstrap tools through the docs, so may be it's okay to leave this as is for this PR, but thought I'd make sure we all start thinking about it.

File: _data/guides.yml [hunk 3]
```diff
@@ -60,6 +60,8 @@ toc:
     path: /docs/user-guide/working-with-resources/
   - title: Garbage collection
     path: /docs/user-guide/garbage-collector/
+  - title: Using NetworkPolicy
```
Comment (errordeveloper) [inline _data/guides.yml]: I think there should be Calico in the title? I'd be really great to have a general intro page talking about policy API and how it works from user's point of view, that page should also link to design documents and whatever resources we have on the (SIG Slack channel, the mailing list etc), and implementation guides including this one.

Comment (chrismarino) [inline _data/guides.yml]: I agree. This is great info, but specific to the Calico implementation.

**Comment by devin-donnelly** [comment]
Please address the open comments from @errordeveloper and @chrismarino before I'll merge.

File: docs/getting-started-guides/network-policy.md (hunk 2 above)
Comment (caseydavenport) [inline docs/getting-started-guides/network-policy.md]: Yep, good to point it out.  I'd very much like the docs to use the next-gen provisioning, but I think it should be done as part of the the update to propagate the new tools into the documentation.

**Comment by caseydavenport** [comment]
@errordeveloper @chrismarino @devin-donnelly Thanks for the feedback, should be all set now :)

**Comment by caseydavenport** [comment]
Closing, will re-open against master.

**Pull request closed without merging**